tools/html-grep

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+from __future__ import absolute_import
+from __future__ import print_function
+from lib.html_grep import grep
+import optparse
+import sys
+
+from six.moves import filter
+try:
+    import lister
+except ImportError as e:
+    print("ImportError: {}".format(e))
+    print("You need to run the Zulip linters inside a Zulip dev environment.")
+    print("If you are using Vagrant, you can `vagrant ssh` to enter the Vagrant guest.")
+    sys.exit(1)
+
+def check_our_files():
+    # type: () -> None
+    parser = optparse.OptionParser()
```

Comment:
Would add usage either explaining what this does or linking to a documentation page that explains what it does, since it's not going to be clear to most people what its purpose is